packages/next/export/index.ts

```diff
@@ -93,8 +93,10 @@ export default async function(
   const nextConfig = configuration || loadConfig(PHASE_EXPORT, dir)
   const threads = options.threads || Math.max(cpus().length - 1, 1)
   const distDir = join(dir, nextConfig.distDir)
-  if (!options.buildExport) {
-    const telemetry = new Telemetry({ distDir })
+
+  let telemetry = options.buildExport ? null : new Telemetry({ distDir })
```

Comment:
```suggestion
  const telemetry = options.buildExport ? null : new Telemetry({ distDir })
```
We shouldn't allow re-assigning this right?